script/update_containers.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+
+set -e -u
+
+SCRIPT_NAME=$(basename "$0")
+
+HELP=$(cat <<EOH
+Usage: $SCRIPT_NAME <prod|beta|test> [on <hosts list>]"
+
+Update MusicBrainz website/webservice containers on specified hosts.
+If no (space-delimited) hosts list is specified, update on all hosts.
+EOH
+)
+
+MB_SERVER_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")/../" && pwd)
+
+cd "$MB_SERVER_ROOT"
+
+if [ $# -eq 0 ]
+then
+  echo >&2 "$SCRIPT_NAME: missing arguments"
+  echo >&2 "$HELP"
+  exit 64
+elif echo "$1" | grep -Eqvx 'prod|beta|test'
+then
+  echo >&2 "$SCRIPT_NAME: unrecognized argument: $1"
+  echo >&2 "$HELP"
+  exit 64
+fi
+
+DEPLOY_ENV=$1
+shift
+
+SERVICES="musicbrainz-webservice musicbrainz-website"
+
+if [ $# -eq 0 ]
+then
+  LIST_METABRAINZ_HOSTS=${LIST_METABRAINZ_HOSTS:-../docker-server-configs/scripts/list_nodes.sh}
+  if ! type "$LIST_METABRAINZ_HOSTS" >/dev/null
```

Comment:
Would `-f` work here? I'm confused why `type` is used instead (though I'm not a bash expert).

Reply:
Yes, actually that would be even more reliable.

(The intent was to match alias too, but alias would not be substituted if set in a variable anyway.)

Reply:
Replaced in https://github.com/metabrainz/musicbrainz-server/pull/1337/commits/e0db1173ea4fd1f6a8a356044417031f08570477